examples/wandb_sweeps/README.md

```diff
@@ -0,0 +1,5 @@
+# W&B Sweeps
+
+Example scripts for making a W&B sweep, training agents in the sweep space, and downloading the results. Read about W&B sweeps [here](https://docs.wandb.ai/guides/sweeps).
+
```

Comment:
If you have pandoc, can you autowrap these lines? I do it like so:

`pandoc -f markdown -t markdown --columns 80 < foo > bar`